util/response_request.py

```diff
@@ -0,0 +1,30 @@
+from util.str_utils import preprocess_text
+import requests
+
+
+def request_hero_name_set():
+    WEB_REQUEST = requests.get('https://dota2.gamepedia.com/api.php?' +
+                                'action=cargoquery&tables=heroes&fields=title&where=game'+
+                                '+IS+NULL&limit=500&format=json')
+    WEB_JSON = WEB_REQUEST.json()
+    SELECTED_HERO_RESPONSES = {}
+    HERO_NAME_RESPONSES = set()
+    for WEB_OBJECT in WEB_JSON['cargoquery']:
+        HERO_ENTRIES = preprocess_text(WEB_OBJECT['title']['title'])
+        if HERO_ENTRIES not in SELECTED_HERO_RESPONSES: 
+            HERO_NAME_RESPONSES.add(HERO_ENTRIES)
+    return (HERO_NAME_RESPONSES)
```

Comment:
` return (HERO_NAME_RESPONSES) ` is same as ` return HERO_NAME_RESPONSES `, so the extra paranthesis can be dropped.

Reply:
Fixed in https://github.com/Jonarzz/DotaResponsesRedditBot/pull/41/commits/c370f726561bfe9b60a2ec63a0d868aa89905b8a.